styles.css

```diff
@@ -408,6 +428,110 @@ textarea {
   color: var(--clr-white);
 }
 
+/* PopUp Window */
+.feature {
+  background-color: #fff;
+  width: 60%;
+  height: 95%;
+  padding: 0 2rem;
+  overflow-y: scroll;
+  border-radius: 16px;
+}
+
+.overlay-content {
+  width: 100%;
+  height: 100%;
+  background-color: rgba(0, 0, 0, 0.7);
+  position: fixed;
+  top: 0;
+  z-index: 10;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.disable {
+  display: none;
+}
+
```

Comment:
- [ ] Kindly make sure that the background of the popup should be blurred and transparent as per the attached loom video.
- [ ] Kindly make sure to display a light gray colour horizontal line after tech stack in the mobile popup as per the attached loom video.
- [ ] Kindly make sure to centre buttons in the mobile popup as per the attached loom video.